SCSH_3C_iOS/SCSH_3C_iOSTests/Repository/BoardRepositoryTests.swift

```diff
@@ -0,0 +1,232 @@
+    ///                                                                                      action: `"https://example.open_product2"`)`,
+    ///                                                                   Persistence.Board(id: `456`,
+    ///                                                                                     code: `"explore_carousel_board"`,
+    ///                                                                                     image-url: `"https:example.example2.jpg"`,
+    ///                                                                                     action-type: `"web_link"`,
+    ///                                                                                     action: `"https://example.open_product2"`)`]`
+    ///                                        saveAllResult: .success(`()`)
+    ///
+    ///
+    func testfetchExploreBoardSuccess() {
+        // TODO: impelement after Api is ready.
+    }
+    
+    private func makeSUT(boardCoreDataService: BoardCoreDataServiceType, moyaNetworkFacade: MoyaNetworkFacadeType, mapper: ExploreBoardMapperType) {
+       sut = BoardRepository(boardCoreDataService: boardCoreDataService, moyaNetworkFacade: moyaNetworkFacade, mapper: mapper)
+    }
+    
+    private func makeMockCoreDataModel(id: String, imageURL: String, actionType: String, action: String) -> Persistence.Board {
+        let board = Persistence.Board(context: CoreDataController.shared.container.viewContext)
+        board.id = Int64(id) ?? 0
+        board.code = "explore_carousel_board"
+        board.image_url = URL(string: imageURL)
+        board.action_type = actionType
+        board.action = action
+        
+        return board
+    }
+    
+}
+
+class MockBoardCoreDataService: BoardCoreDataServiceType {
+    
+    private let getResult: Result<[Persistence.Board], Error>
+    private let saveResult: Result<Void, Error>
+    private let saveAllResult: Result<Void, Error>
+    
+    init(getResult: Result<[Persistence.Board], Error>, saveResult: Result<Void, Error>, saveAllResult: Result<Void, Error>) {
+        self.getResult = getResult
+        self.saveResult = saveResult
+        self.saveAllResult = saveAllResult
+    }
+    
+    func get(code: String?) -> AnyPublisher<[Persistence.Board], Error> {
+        Future { promise in
+            promise(self.getResult)
+        }
+        .eraseToAnyPublisher()
+    }
+    
+    func save(board: Persistence.Board) -> AnyPublisher<Void, Error> {
+        Future { promise in
+            promise(self.saveResult)
+        }
+        .eraseToAnyPublisher()
+    }
+    
+    func saveAll(boards: [Persistence.Board]) -> AnyPublisher<Void, Error> {
+        Future { promise in
+            promise(self.saveAllResult)
+        }
+        .eraseToAnyPublisher()
+    }
+}
+
+class MockMoyaNetworkFacade_Board: MoyaNetworkFacadeType {
```

Comment:
What's the class naming convention in Swift?

Reply:
> What's the class naming convention in Swift?

`upper camel case`

but something I'll make an exception if I extremely want to emphasize the usage of the object, I'll add the term at the end of the name. 

For instance, 
``` swift
func execute_iOS16 {}
func execute {}
``` 

In my situation, many unit tests require the creation of a Mock Object for `MockMoyaNetworkFacade`,  Since it is designed with generics,  I add the term  _{Model} to emphasize the specific usage of the object.

Do you think it's a good practice?